pyro/infer/mcmc/mcmc.py

```diff
@@ -257,10 +257,10 @@ def __init__(self, kernel, num_samples, warmup_steps=0,
                               .format(num_chains, available_cpu))
                 num_chains = available_cpu
         if num_chains > 1:
-            self.sampler = _ParallelSampler(kernel, num_samples, warmup_steps,
+            self.sampler = _ParallelSampler(kernel, num_samples, self.warmup_steps,
```

Comment:
this is a bug when `warmup_steps` is None, we should use `self.warmup_steps` here.